lib/get-urls-from-file.js

```diff
@@ -0,0 +1,8 @@
+const { readFileSync } = require('fs')
+
+module.exports = path => {
+  const fileType = path.toLowerCase().split('.').pop()
+  const data = readFileSync(path, 'utf-8').toString()
+  console.log(`\nGot urls from the ${fileType} file at ${path}`)
+  return fileType === 'json' ? JSON.parse(data) : data.split('\n')
+}
```

Comment:
I think this looks fine.
Once we merge and close this pull request, I can add the xml parsing so we have one more option for sitemaps

Reply:
I can also make a ```get-urls-from-hosted-file``` and include that in ```/lib```, where if people have hosted json/txt on their webserver say at https://alheimsins.net/test-urls.txt, it will still work.

Reply:
I wonder if this will be an issue, using the ```html-validator-cli```, I ran the following:
```$ html-validator http://asdf```
and it said the page is valid.
This means if I give it a test txt file that actually does not contain any valid urls like the following
```
random.string.1
random.random.random
```
It will actually run parse each of these as if they are url and return that they are valid...

This is what I meant:
![image](https://user-images.githubusercontent.com/22629067/53111487-24cf7980-350b-11e9-8f4f-b1b48e4b30e2.png)

the protocols would be added via the normalizer

Reply:
Hmm, strange

File an issue? I'll have a talk with the maintainer of that module ;-)

Reply:
ok, I think the issue is smaller than what I thought. There's nothing wrong with html-validator per se.
I tried just the following code
```JavaScript
(async () => {
  const validator = require('html-validator')
  let data = await validator({url: 'http://asdf'})
  console.log(data)
})()
```
and it outputs
```
{"url":"http://asdf","messages":[{"type":"non-document-error","subType":"io","message":"asdf: No address associated with hostname"}]}
```
This is the correct output.

This means the ```Page is valid``` thing is just a slight issue with ```html-validator-cli``` which I'll open an issue and link here

As for what needs to be done here, we have to make a decision on whether to do a preliminary url validation, or just let w3c validator api handle it. I think this depends on whether making requests to w3c validator is costly (for example, if there's an hourly quota from an IP address, I don't think there is btw)

Reply:
the w3c quota is something like "fair use" if I remember correct.

Reply:
For the scope of this pull request, let's just go with the latter then.
I think validation beforehand should be considered as another feature.